plugins/woocommerce/src/Blocks/BlockTypes/MiniCartCartButtonBlock.php

```diff
@@ -27,11 +27,28 @@ protected function render_experimental_iapi_markup( $attributes, $content, $bloc
 		$view_cart_text         = $attributes['cartButtonLabel'] ? $attributes['cartButtonLabel'] : $default_view_cart_text;
 		$cart_page_id           = wc_get_page_id( 'cart' );
 		$cart_page_url          = get_permalink( $cart_page_id );
+		$classes                = implode(
+			' ',
+			array_filter(
+				array(
+					'wc-block-components-button',
+					'wp-element-button wc-block-mini-cart__footer-cart',
+					// Default style class is not added by default, so it needs to be added manually if it doesn't exist.
+					( ! isset( $attributes['className'] ) || strpos( $attributes['className'], 'is-style-' ) === false ) ? 'is-style-outline' : '',
```

Comment:
Sharing this related comment after modifying the code: https://github.com/woocommerce/woocommerce/pull/59561#discussion_r2195272901